functions/src/triggers/storage.ts

```diff
@@ -0,0 +1,45 @@
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
+import { ObjectMetadata } from 'firebase-functions/lib/providers/storage';
+import * as sharp from 'sharp';
+
+import { ImageData } from '../../../models/image-data.models';
+
+export const convertToWebP = functions.storage.object().onFinalize(async (object: ObjectMetadata) => {
```

Comment:
I think that the name `convertAndResizeImage` of the function below is a better name that is less specific to its implementation choices.

Reply:
I've change the function name in 6a28be9